scripts/install/usr/lib/systemd/system/m0d@.service

```diff
@@ -30,4 +30,5 @@ TimeoutStopSec=5min
 Restart=no
 Environment=SHELL=/bin/bash
 Group=motr
-ExecStart=/usr/libexec/cortx-motr/motr-server m0d-%i
+#ExecStart=/usr/libexec/cortx-motr/motr-server m0d-%i
+ExecStart=/usr/libexec/cortx-motr/motr-start m0d-%i
```

Comment:
Why we need this change ?

Reply:
During CFT testing in one of the issue, took more than 5 mins to shutdown the process.
Made it generic, so that this commit can be used in both main and kubernetes branches.

Oct 27 00:44:50 ssc-vm-rhev4-0857.colo.seagate.com systemd[1]: Stopping m0d helper for 0x7200000000000001:0xa service-FID...

Oct 27 00:49:50 ssc-vm-rhev4-0857.colo.seagate.com systemd[1]: Stopped m0d helper for 0x7200000000000001:0xa service-FID.